backend/server/public/src/admin.jsx

```diff
@@ -322,34 +363,31 @@ let Login = ({ dispatch }) => {
 };
 Login = connect()(Login);
 
+let LoggedInApp = () => {
+  return (<div>
+    <Route exact path="/facebook-ads/admin" component={Ads} /> {/* confusingly, despite being `exact`, this matches /facebook-ads/admin, without the trailing slash */}
+    <Route exact path="/facebook-ads/admin/ads" component={Ads} />
+    <Route exact path="/facebook-ads/admin/ads/" component={Ads} />
```

Comment:
Looks like it might? It uses https://github.com/pillarjs/path-to-regexp on that `path` property. As described above, trailing slashes on `exact` matches are optional, so I removed the top one (without the trailing slash) and it still works.

Reply:
Ah cool. Probably could remove the second one here "/facebook-ads/admin/ads/"